src/openrct2/world/Climate.cpp

```diff
@@ -140,37 +139,39 @@ void ClimateUpdate()
         }
         else if (!(GetGameState().CurrentTicks & 0x7F))
         {
-            if (gClimateCurrent.Temperature == gClimateNext.Temperature)
+            auto& gameState = GetGameState();
```

Comment:
Look two lines higher the ga estate is already being used there. Maybe follow Matt's suggestion and move this back to where it was and use the reference in the if too.

Reply:
Ok I have applied the changes.